multipatch_analysis/ui/acq4_modules.py

```diff
@@ -142,6 +140,13 @@ def create_lims_json(self):
         Gets timestamp
         Creates dictionary of cell locations and enables the save button
         """
+        
+        if hasattr(self, 'specimen_name') is False:
+            self.base_dir = self.mosaic_editor.ui.fileLoader.baseDir()
```

Comment:
Don't copy code--I would just make a `specimen_name` property to handle this

Reply:
yeah I was thinking about that